WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+QuickActions.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+extension BlogDetailsViewController {
+    @objc func quickActionsSectionViewModel() -> BlogDetailsSection {
+        return BlogDetailsSection(title: nil,
+                                  rows: [BlogDetailsRow()],
+                                  footerTitle: nil,
+                                  category: .quickAction)
+    }
+}
+
+@objc class QuickActionsCell: UITableViewCell {
+    private var actionRow: ActionRow!
+
+    func configure(with items: [ActionRow.Item]) {
+        guard actionRow == nil else {
+            return
+        }
+
+        actionRow = ActionRow(items: items)
+        contentView.addSubview(actionRow)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        actionRow.translatesAutoresizingMaskIntoConstraints = false
+
+        let widthConstraint = actionRow.widthAnchor.constraint(equalToConstant: 350)
+        widthConstraint.priority = .defaultHigh
+
+        NSLayoutConstraint.activate([
+            actionRow.topAnchor.constraint(equalTo: contentView.topAnchor),
+            actionRow.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            actionRow.leadingAnchor.constraint(greaterThanOrEqualTo: contentView.leadingAnchor),
+            actionRow.trailingAnchor.constraint(lessThanOrEqualTo: contentView.trailingAnchor),
+            actionRow.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
```

Comment:
While I realize these constraints were copied from the existing constraints in BlogDetailHeaderView, I'm pretty sure you can delete the trailing anchor. 

The combination of the leading anchor and center x  anchor should be enough to position the action row.

Reply:
I tested deleting that constraint, and it works as expected. I leave the decision up to you though!

Reply:
Fixed in c4e33807ef07d05c4b481a23d5069c608faabcfa